sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -132,6 +133,11 @@ selectItem
     | ASTERISK                                                                       #selectAll
     ;
 
+returning
+    : RETURNING selectItem (',' selectItem)*
```

Comment:
But this will allow `RETURNING *, *, *` right, it is not what we documented. Does pg supports it? Also, `schema_name.*` as a select item is missing from the documentation.

Also, if `*, *, *` case is supported, then the grammar can be simplified. The `selectItem` rule already handles ASTERISK, therefore ASTERISK in the `returning` is not needed.

Reply:
Good point, pg actually does support `RETURNING *, *, *` therefore the ASTERISK can be dropped.